[CI] test failure SnapshotStatusApisIT.testStatusAPICallInProgressSnapshot

Test failed on 7.x. Doesn't reproduce for me.

./gradlew :server:integTest --tests "org.elasticsearch.snapshots.SnapshotStatusApisIT.testStatusAPICallInProgressSnapshot" \
  -Dtests.seed=D0C1B4242C6B7FA9 \
  -Dtests.security.manager=true \
  -Dtests.locale=es-CU \
  -Dtests.timezone=Etc/GMT+4 \
  -Dcompiler.java=12 \
  -Druntime.java=8

Log: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+multijob-unix-compatibility/os=oraclelinux-6&&immutable/219/console
Build Scans: https://gradle-enterprise.elastic.co/s/5cnjuw2u5gsnc

```
java.lang.AssertionError: expected:<INIT> but was:<STARTED>
```

--- a/server/src/test/java/org/elasticsearch/snapshots/SnapshotStatusApisIT.java
+++ b/server/src/test/java/org/elasticsearch/snapshots/SnapshotStatusApisIT.java
@@ -29,6 +29,7 @@ import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.common.unit.TimeValue;
 
 import java.util.List;
+import java.util.concurrent.TimeUnit;
 
 import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked;
 import static org.hamcrest.Matchers.equalTo;
@@ -76,7 +77,7 @@ public class SnapshotStatusApisIT extends AbstractSnapshotIntegTestCase {
         assertEquals(snStatus.getStats().getTime(), snapshotInfo.endTime() - snapshotInfo.startTime());
     }
 
-    public void testStatusAPICallInProgressSnapshot() throws InterruptedException {
+    public void testStatusAPICallInProgressSnapshot() throws Exception {
         Client client = client();
 
         logger.info("-->  creating repository");
@@ -101,7 +102,7 @@ public class SnapshotStatusApisIT extends AbstractSnapshotIntegTestCase {
 
         final List<SnapshotStatus> snapshotStatus = client.admin().cluster().snapshotsStatus(
             new SnapshotsStatusRequest("test-repo", new String[]{"test-snap"})).actionGet().getSnapshots();
-        assertEquals(snapshotStatus.get(0).getState(), SnapshotsInProgress.State.STARTED);
+        assertBusy(() -> assertEquals(SnapshotsInProgress.State.STARTED, snapshotStatus.get(0).getState()), 1L, TimeUnit.MINUTES);
 
         logger.info("--> unblock all data nodes");
         unblockAllDataNodes("test-repo");

